src/ctoken.ts

```diff
@@ -204,6 +217,18 @@ export function handleRepayBorrow(event: RepayBorrow): void {
     createAccount(accountID)
   }
 
+  if (event.params.accountBorrows.equals(zeroBI)) {
```

Comment:
`accountBorrows` should always be a BigInt 0 if the loan is fully repaid, so we should decrement `market.borrowerCount`